osf/models/preprint_provider.py

```diff
@@ -28,6 +28,7 @@ class PreprintProvider(ObjectIDMixin, BaseModel):
     social_twitter = models.CharField(null=True, blank=True, max_length=200)  # max length on prod: 8
     social_facebook = models.CharField(null=True, blank=True, max_length=200)  # max length on prod: 8
     social_instagram = models.CharField(null=True, blank=True, max_length=200)  # max length on prod: 8
+    share_source = models.CharField(blank=True, max_length=200)
```

Comment:
There should be a list or similar in the Admin to keep certain fields from being exported with the preprint_provider records. Let's update the admin to make sure that this field is not exported.